cmd/tink-worker/container/container.go

```diff
@@ -0,0 +1,104 @@
+package container
+
+import (
+	"context"
+	"path"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/pkg/errors"
+)
+
+type containerConfigData struct {
+	pid            string
+	env            []string
+	containerName  string
+	volumeBindings []string
+	cmd            []string
+	imageName      string
+	registry       string
+}
+
+// create creates container and returns containerid on success else returns error.
+func create(ctx context.Context, cli client.ContainerAPIClient, configData containerConfigData, captureLogs bool) (string, error) {
```

Comment:
should we move `captureLogs` into `configData`?

Reply:
All vars in configData are used to creat container.   but captureLogs var will be more external to that process?

Reply:
is that an implementation detail of this `func create`? as a consumer of this func, is there a difference? there are trade-offs either way, curious about the intent.

Reply:
> is that an implementation detail of this `func create`? as a consumer of this func, is there a difference? there are trade-offs either way, curious about the intent.

For usability that var can be in configData but readability, I think separate var make sense?